src/bosh_aws_cpi/spec/unit/instance_param_mapper_spec.rb

```diff
@@ -579,11 +582,10 @@ module Bosh::AwsCloud
           end
 
           it 'attaches an ipv4 and an ipv6 to the nic' do
-            allow(logger).to receive(:info)
+            allow(logger).to receive(:warn)
```

Comment:
```suggestion
```
Does this case actually raise a warning? If these should have the same subnet id I think it should not warn

Reply:
yes, you are right, it does not